ChangeLog.md

```diff
@@ -33,12 +41,31 @@
 * Settings tab layout has been redesigned to support longer texts
 * Added button to start map editor directly from Launcher
 * Simplified game starting flow from online lobby
+* Mod description will now show list of languages supported by mod
+* Launcher now uses separate mod repository from vcmi-1.1 version to prevent mod updates to unsupported versions
 
 ### AI PLAYER:
+* Nullkiller AI is now used by default
 * AI should now be more active in destroying heroes causing treat on AI towns
 * AI now has higher priority for resource-producing mines
 * Increased AI priority of town dwelling upgrades
 * AI will now de-prioritize town hall upgrades when low on resources
+* Messages from cheats used by AI are now hidden

```

Comment:
* Improved army gathering from towns., AI will also attempt to exchange armies between main heroes to get the strongest hero with the strongest army.
* Improved Pandora handling
* AI takes into account fort level now when evaluating enemy town capturing priority.

Reply:
* AI can not use allied shipyard now to avoid freeze

in progress
* AI will avoid attacking creatures standing on draw-bridge tile during siege if the bridge is closed.
* AI will consider retreat during siege if it can not do anything (catapult is destroyed, no destroyed walls exist)